polaris.shopify.com/content/foundations/content/naming.md

```diff
@@ -81,16 +81,15 @@ Only Shopify can use the word “Shopify” in a name. The word “Shopify” ca
 It’s important we use the word “Shopify” consistently, and sparingly. Don’t use
 “Shopify” in a name unless there’s a lack of surrounding context and we want its target audience to know it’s associated with Shopify.
 
-Adding “Shopify” doesn’t add clarity in the context of other Shopify products and features. For example, merchants often confuse Shopify Shipping with the other shipping features, like shipping settings, carrier calculated shipping other, shipping labels, and shipping zones. Our support staff have to refer to the Shopify Plan by its cost because the general name doesn’t distinguish it ratesthe other plans.
+Adding “Shopify” doesn’t add clarity in the context of other Shopify products and features. For example, merchants often confuse Shopify Shipping with the other shipping features, like shipping settings, carrier calculated shipping rates, shipping labels, and shipping zones. Our support staff have to refer to the Shopify Plan by its cost because the general name doesn’t distinguish it from the other plans.
 
 Use “Shopify” in front of a name when a product:
 
 - Is or will become a separate product or platform and we need to associate it with Shopify
 - Should be differentiated from other, similar products in the industry
-- Doesn’t justify an
-  [evocative name](/content/naming#section-descriptive-vs-evocative-names), but still needs to associated with Shopify
+- Doesn’t justify an [evocative name](/content/naming#section-descriptive-vs-evocative-names), but still needs to associated with Shopify
```

Comment:
```suggestion
- Doesn’t justify an [evocative name](/content/naming#descriptive-vs-evocative-names), but still needs to associated with Shopify
```

Ran it locally and looks like links to other markdown sections don't work if you include `section-`